### ISSUE TITLE
Test suite fails (with Mojolicious 6.17?)

### ISSUE THREAD

**Issue by eserte** [reneeb/Mojolicious-Plugin-FormFieldsFromJSON#2]
Test log looks like this

```
#   Failed test 'exact match for content'
#   at t/fields/05_select_preselect.t line 55.
#          got: '<select id="language" name="language"><option value="cn">cn</option><option value="de">de</option><option value="en">en</option></select>'
#     expected: '<select id="language" name="language"><option value="cn">cn</option><option selected="selected" value="de">de</option><option value="en">en</option></select>'

#   Failed test 'exact match for content'
#   at t/fields/05_select_preselect.t line 63.
#          got: '<select id="language" name="language"><option value="cn">cn</option><option value="de">de</option><option value="en">en</option></select>'
#     expected: '<select id="language" name="language"><option value="cn">cn</option><option selected="selected" value="de">de</option><option value="en">en</option></select>'

#   Failed test 'exact match for content'
#   at t/fields/05_select_preselect.t line 79.
#          got: 'en<select id="language" name="language"><option value="cn">cn</option><option value="de">de</option><option value="en">en</option></select>'
#     expected: 'en<select id="language" name="language"><option value="cn">cn</option><option selected="selected" value="de">de</option><option value="en">en</option></select>'
# Looks like you failed 3 tests of 15.
t/fields/05_select_preselect.t ......................... 
Dubious, test returned 3 (wstat 768, 0x300)
Failed 3/15 subtests 
```

(etc., more stuff with missing "selected" or "checked")

Statistical analysis shows that the failures happen with Mojolicious 6.17, but not with 6.11 or earlier.

**Comment by reneeb** [comment]
Thanks! Fixed with version 0.31

**Comment by eserte** [comment]
This seems to work with newer Mojolicious, but now it fails with older Mojolicious, e.g. 5.79. Maybe you want to increase the minimum version or skip the test in case of old Mojolicious?
